scripts/background.js

```diff
@@ -460,6 +484,50 @@ chrome.runtime.onMessage.addListener(function(message,sender,sendResponse){
                 chrome.tabs.sendMessage(tabs[0].id, {RTurl:RTurl});
                 console.log(RTurl);
             });
+        }else if(message.message=='changeBadge'){
+          var tabId=message.tabId;
+          console.log(tabId);
+          chrome.browserAction.setBadgeText({tabId: tabId, text:"\u2713"});
+        }else if(message.message=='showall'){
+          chrome.storage.sync.get(['show_context'],function(event){
+            if(!event.show_context){
+              event.show_context="tab"; //By-default the context-window open in tabs
+            }
+            var received_url=message.url; //URL which is received by message-parsing_url
+            received_url = received_url.replace(/^https?:\/\//,'');
+            var length=received_url.length; 
+            for(var i=0;i<length;i++){
+              if(received_url[i]=='/'){
+                  last_index=i;
+                  break;
+              }
+            }
```

Comment:
@anishsarangi I found something else.
There is a much easier way to find the location of a character in a string `url.indexOf('/')`. Please use it here.